web/client/translations/data.en-US.json

```diff
@@ -31,6 +31,12 @@
         "version": {
             "label": "Version"
         },
+        "errorPage": {
+            "title":"Ops, Something has gone wrong",
+            "subtitle":"Please click the icon below to reload MapStore",
+            "description":"if you want to report this error you can send an email our ",
```

Comment:
You should not combine localized strings (order of words depends on the language). 
I suggest to make mailing list/administrator link optional  because the "our mailing list" is valid only for the product, not for customizations.

I suggest more like A generic `please contact the administrator` without any configuration. 
Using ConfigIUtils to get from (localConfig)  `administratorAddress` or  `mailinglistAddress`, you can be more specific:
please contact the administrator <a href=mailto:.../>
please contact the mailing list at <a href=mailto:.../>.